docs/README.md

```diff
@@ -0,0 +1,9 @@
+# Kamaji documentation
+
+- [Architecture](./architecture.md)
+- [Concepts](./concepts.md)
+- [Getting started](./getting-started-with-kamaji.md)
+- [Deployment](./kamaji-deployment-guide.md)
+  - [Tenant deployment](./kamaji-tenant-deployment-guide.md)
+  - [Deployment on cloud provider: Azure](./kamaji-azure-deployment-guide.md)
+- [Reference](./reference.md)
```

Comment:
I would suggest this structure:

# Kamaji documentation

- [Concepts](./concepts.md)
- [Architecture](./architecture.md)
- [Getting started on KinD](./getting-started-with-kamaji.md)
- [Kamaji Deployment](./kamaji-deployment-guide.md)
- [Tenant deployment](./kamaji-tenant-deployment-guide.md)
- Deployment on Cloud Providers:
   - [Azure](./kamaji-azure-deployment-guide.md)
- [Reference](./reference.md)

Reply:
Applied